core/store/src/flat/types.rs

```diff
@@ -4,10 +4,35 @@ use near_primitives::hash::CryptoHash;
 use near_primitives::state::ValueRef;
 use near_primitives::types::BlockHeight;
 
+/// Defines value size threshold for flat state inlining.
+/// It means that values having size greater than the threshold will be stored
+/// in FlatState as `FlatStateValue::Ref`, otherwise the whole value will be
+/// stored as `FlatStateValue::Inlined`.
+/// See the following comment for reasoning behind the threshold value:
+/// https://github.com/near/nearcore/issues/8243#issuecomment-1523049994
+pub const INLINE_DISK_VALUE_THRESHOLD: usize = 4000;
+
 #[derive(BorshSerialize, BorshDeserialize, Debug, Clone, PartialEq, Eq)]
 pub enum FlatStateValue {
     Ref(ValueRef),
-    // TODO(8243): add variant here for the inlined value
+    Inlined(Vec<u8>),
```

Comment:
I have a little doubt if we should introduce it as a term in codebase. But it makes sense because our default behaviour is "not-inlined".